src/groupeddataframe/grouping.jl

```diff
@@ -190,42 +248,76 @@ end
 
 """
     combine(gd::GroupedDataFrame)
-    combine(f::Function, gd::GroupedDataFrame)
+    combine(cols => f, gd::GroupedDataFrame)
+    combine(f, gd::GroupedDataFrame)
 
 Transform a `GroupedDataFrame` into a `DataFrame`.
-If a function `f` is provided, it is called for each group in `gd` with a `SubDataFrame` view
-holding the corresponding rows, and the returned `DataFrame` then consists of the returned rows
-plus the grouping columns.
+
+If the first argument is a `cols => f` pair, `cols` must be a column name or index, or
```

Comment:
If I understand it correctly `combine` can also ingest a named tuple as a first argument. Right?

Reply:
Did you leave this pattern as accepted or drop it?

Reply:
I'm not sure what to do about that. It's supported as a by-product of accepting `f::Any` in first position. I could write a special method just to throw an error, but that sounds a bit absurd. So I've added a mention that this is supported.

Reply:
OK